.eslintignore

```diff
@@ -0,0 +1,11 @@
+**/node_modules/**
+**/dist/**
+**/dist-types/**
+**/storybook-static/**
+**/coverage/**
+**/build/**
+**/.git/**
+**/public/**
+playwright.config.ts
+.eslintrc.js
+workspaces
```

Comment:
consider setting the editor to automatically adding newlines :)

Interestingly, we DO want the license header check to run on all workspaces right? But we'll make sure to add that everywhere later then?